### ISSUE TITLE
Cognito sign-on using Mosaic auth

### ISSUE THREAD

**Issue by tonydisera** [iobio/clin.iobio#58]

**Comment by chmille4** [comment]
is this aws cognito?

**Comment by tonydisera** [comment]
Yes.  I was wondering if we could use an Identity pool that would connect to Mosaic Auth as an 'External Identity Provider'.  This would get us through the CSER demo until we can save the analysis results in Mosaic.
